tests/system/providers/google/cloud/compute/example_compute.py

```diff
@@ -43,12 +42,12 @@
 from tests.system.providers.google import DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
 
 # [START howto_operator_gce_args_common]
-ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
-PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT") or DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID", "default")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT", "default") or DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
```

Comment:
In this implementation we would never have the option after the `or` statement, because `os.environ.get()` always returns `default` when `SYSTEM_TESTS_GCP_PROJECT` equals `None`. Could you refactor this?